sdk/src/main/java/io/horizen/json/serializer/SignatureVFieldSerializer.java

```diff
@@ -0,0 +1,27 @@
+package io.horizen.json.serializer;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import io.horizen.account.transaction.EthereumTransaction;
+import org.web3j.crypto.Sign;
+
+import java.io.IOException;
+import java.math.BigInteger;
+
+
+public class SignatureVFieldSerializer extends JsonSerializer<BigInteger> {
+    @Override
+    public void serialize(BigInteger val, JsonGenerator jsonGenerator, SerializerProvider serializerProvider) throws IOException {
+        try {
+            Object pojo = jsonGenerator.getOutputContext().getParent().getCurrentValue();
+            if (pojo instanceof EthereumTransaction && ((EthereumTransaction) pojo).isLegacy())
```

Comment:
Maybe we should test `isEIP1559()` eth tx attribute, after all we have 3 alternatives for the tx type (0, 1, 2), even if 1 is currently not used in EON but it is supported by geth and in future maybe we will also use that type

Reply:
agree! 
also I suggest to copy&paste here the same comment in the test
// isEIP1559 txs are defined to use 0 and 1 ..... 
just to remember in the future why we did this